src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-list/user-list.component.ts

```diff
@@ -99,8 +99,8 @@ export class UserListComponent implements OnInit {
     if (sessionUsername === username) {
       this.notificationService.show(
         NotificationType.error,
-        `You are currently authenticated with user "${username}".`,
-        'Cannot Delete User'
+        'Cannot Delete User',
```

Comment:
Could you please use the wording of task messages here? (Also noted in `hacking.rst`) =>
`Failed to delete user ${username}`